catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -128,7 +128,9 @@ public static Metacard createMetacard(
         Contact.POINT_OF_CONTACT_NAME,
         metadata.get(Office.USER_DEFINED_METADATA_NAME_PREFIX + "owner"));
 
-    setAttribute(metacard, Contact.CONTRIBUTOR_NAME, metadata.get(Office.LAST_AUTHOR));
+    if (!metadata.get(Office.LAST_AUTHOR).equals(metadata.get(TikaCoreProperties.CREATOR))) {
```

Comment:
❓ Are these things null safe? (will `metadata.get(Office.LAST_AUTHOR)`  always be non null?

Reply:
Ah good catch. I see that we are null checking in the `setAttribute` method, so these should probably be null checked here.